src/vouchers.js

```diff
@@ -165,11 +174,7 @@ class RecrasVoucher {
         this.findElement('.buyTemplate').addEventListener('click', this.buyTemplate.bind(this));
     }
 
-    showContactForm(templateId) {
-        this.selectedTemplate = this.templates.filter(t => {
-            return t.id === templateId;
-        })[0];
-
+    showContactForm() {
```

Comment:
If the changes to `showContactForm` belong in this PR, wouldn't it be more appropriate to pass `this.selectedTemplate` to `showContactForm()`, to eliminate an implicit coupling via class property `selectedTemplate`?